external-crates/move/crates/move-analyzer/trace-debug/src/extension.ts

```diff
@@ -240,10 +254,16 @@ class MoveTraceViewProvider implements vscode.CustomReadonlyEditorProvider {
         webviewPanel: vscode.WebviewPanel,
         _token: vscode.CancellationToken
     ) {
+        if (document.uri.scheme === TRACE_FILE_URI_SCHEME) {
```

Comment:
When explicitly choosing to open compressed trace file in the default text editor, VSCode closes the original (compressed) file and opens a new one (with a different URI scheme) which is then post-processed by custom content provider. However, if the default editor is custom Move Trace View editor, it will attempt to open another window here to show trace even though we are already displaying it properly. Hence, if we see `mtrace` URI scheme we shouldn't do anything